src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java

```diff
@@ -194,4 +194,8 @@ public void verifyMessageAboutNewPassword(String userName) {
 			true
 		);
 	}
+
+	public void verifyFBButtonVisible() {
```

Comment:
This ```verifyFBButtonVisible()``` here, as well as the other ```verifyFBButtonVisible()``` methods found inside SignUpPageObject.java, SpecialUserLoginPageObject.java, SpecialUserLoginPageObject.java, and PreferencesPageObject.java actually aren't necessary.

The reason is that all of these classes inherit from the WikiBasePageObject.java class. Since we have that method defined there, all other classes which inherit from it have that method as well. So you can go ahead and remove all these extra ```verifyFBButtonVisible()``` definitions except for the one found in WikiBasePageObject.java.